myapp/config/routes.rb

```diff
@@ -1,4 +1,15 @@
 Rails.application.routes.draw do
+  # login page
+  get 'login', to: 'sessions#new'
+  # login
+  post 'login', to: 'sessions#create'
+  # logout
+  delete 'logout', to: 'sessions#destroy'
+  # index page
   root 'tasks#index'
+  # signup page
+  get 'users/signup', to: 'users#new'
```

Comment:
"signup" would be more nice, what do you think?

Reply:
Changed the route!
Refer to 76323e1